### PR TITLE
Update JSF Container 

### LINKED ISSUE #15353: Re-enable Stateless ViewScope Mojarra Tests for jsfContainer 2.3 FAT
JSF22StatelessView_TestViewScopeCDIBeanNotTransient_Mojarra
    JSF22StatelessView_TestViewScopeCDIBeanTransient_Mojarra

See https://github.com/OpenLiberty/open-liberty/issues/14533

### PR DESCRIPTION
**Pull request by volosied**
fixes #15353 
fixes # #16028  -- Update to MyFaces 3.0.0 

The  JSF22StatelessView app needed a beans.xml to work properly. I'm not sure why as it was not needed before. I've found that the Mojarra 3.0.0 release is just different, and I can't explain why. 

I also noticed that "org.glassfish:jakarta.faces:3.0.0-RC1" was used as a dependency. It should be RC3.

### CODE REVIEW

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsfContainer_fat_2.3

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsfContainer_fat_2.3

**Comment by volosied** [comment]
If the build in https://github.com/OpenLiberty/open-liberty/pull/14985 is successful, then this PR should be closed.

**Comment by volosied** [comment]
#build

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsfContainer_fat_2.3

File: dev/com.ibm.ws.jsfContainer_fat_2.3/publish/files/beans.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<beans xmlns="http://xmlns.jcp.org/xml/ns/javaee"
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsfContainer_fat_2.3/publish/files/beans.xml:1]: Add copyright header? Also, do we need to set the bean discovery mode here?

File: dev/com.ibm.ws.jsfContainer_fat_2.3/fat/src/com/ibm/ws/jsf/container/fat/tests/JSF22StatelessViewTests.java [hunk 2]
```diff
@@ -12,6 +12,8 @@
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsfContainer_fat_2.3/fat/src/com/ibm/ws/jsf/container/fat/tests/JSF22StatelessViewTests.java:12]: update copyright header date range

**Review by pnicolucci — APPROVED** [review]

**Comment by volosied** [comment]
#build

**Comment by volosied** [comment]
#run-libby-bot

**Pull request merged**